### ISSUE TITLE
[import/default] False positive when parsing a file using ES7.

### ISSUE THREAD

**Issue by doctyper** [benmosher/eslint-plugin-import#36]
When importing a module that contains [stage 0](https://babeljs.io/docs/usage/experimental/) proposals, e.g.:

```
// Foo.jsx
class Foo {
    // ES7 static members
    static bar = true;
}

export default Foo;

// Importer.jsx
import Foo from "Foo"
```

I get the following false-positive:
```
error  No default export found in module  import/default
```

I dug into the code a bit and found Espree has issues parsing experimental code:

```
{ settings: 
   { 'import/ignore': [ 'node_modules' ],
     'import/resolve': { extensions: [Object], moduleDirectory: [Object] } },
  hasDefault: false,
  named: {},
  errors: 
   [ { [Error: Line 6: Unexpected token =]
       index: 185,
       lineNumber: 6,
       column: 24,
       description: 'Unexpected token =' } ] }
```

Note: We use [babel-eslint](https://github.com/babel/babel-eslint) in order to support linting with experimental code.

I'm not asking that you support experimental code (although it would be nice!), but it would be great if the parsing error was surfaced, or at the very least a note that this rule does not support experimental code.

**Comment by benmosher** [comment]
There is a `no-errors` rule (off by default) that exists to warn on this sort of thing.

I just added some explicit tests (with your example file) and the ESLint tests report that `no-errors` should report for this.

**Comment by benmosher** [comment]
I will investigate making `babel-eslint` a parser option. I'm not sure if it can be inferred from the rule context, but if nothing else it could be (re-)specified in an `import/parser` setting.

**Comment by doctyper** [comment]
Thanks! That would be great, the rule is super useful.

**Comment by benmosher** [comment]
Pushed with 0.7.2. Haven't tested significantly, but given that ESLint integrates with babel-eslint in the same way, I have high hopes!

**Comment by doctyper** [comment]
@benmosher I finally had a chance to try this out. This is seriously awesome. It's parsing using `babel-eslint` with flying colors.

If you're ever in L.A. I will gladly take you out for :beers:.

**Comment by benmosher** [comment]
😎
